cypress/objects/logingPage.js

```diff
@@ -0,0 +1,24 @@
+class LoginPage{
+    visit(){
+        cy.visit('/')
+    }
+    getValidationError(){
+        return cy.get('.Select.not-valid')
+    }
+    getSubmit(){
+        return cy.get('[type="submit"]')
+    }
+    getUserForm(){
+        return cy.get('[id="loginForm.userId"]')
+    }
+    getUserOption(userName){
+        return cy.get(`[aria-label="${userName}"]`)
+    }
+    getLoginForm(){
+        return cy.get('[id="loginForm.role"]')
+    }
+    getLabel(role){
```

Comment:
What label is this referring to? Please add some more details to what label/field etc. this is referring.